src/commands/test.js

```diff
@@ -94,6 +96,10 @@ module.exports = {
         'test',
         standaloneNode ? 'docker-compose-standalone-node.yml' : 'docker-compose.yml',
       )
+
+    // parse timeout. Defaults to 10 seconds
+    timeout = Math.abs(parseInt(timeout)) || 10000
```

Comment:
I'm actually ok with defaulting to 100 secs, the default should be what's most likely to work. And the timeout configuration isn't really a breaking change.

Reply:
Or 60 seconds, anyways whatever we are using for our tests.